geometry_msgs/PoseStamped start_pose #start pose for the planner
geometry_msgs/PoseStamped goal_pose #start pose for the planner
---
trajectory_msgs/MultiDOFJointTrajectory sampled_plan
